file.go

```diff
@@ -133,3 +136,47 @@ func TouchFile(f *os.File) error {
 	}
 	return nil
 }
+
+func GetFinalPathName(fileName string, openFlags uint32, finalPathFlags uint32) (result string, err error) {
+	// Todo: GetFinalPathName does not exists on Windows XP. We need other method to get final path name.
+	// Now we use recover() for interoperability with XP
+	defer func() {
+		if e := recover(); e != nil {
+			err = NewWindowsError("GetFinalPathName", errors.New("function call error"))
+		}
+	}()
```

Comment:
Let's remove this.  If Windows XP compatibility is a concern, the best way to handle that would be to use `VersionCheck` in your code calling into gowin32 before calling this function to make sure that we're on a Windows version that supports it.

Reply:
We using this code on Windows XP and this function is called from GetFilesInfo. So without this GetFilesInfo crashes, with this works but only it does not support symlinks.
I know this is not a good solution. I have plan to create GetFinalPathName for XP, but it is a bit complicated. 
So maybe I remove this and change getFilesInfo to something like:
if IsWindowsXP { 
  // something 
} else {}
What do you think?

Reply:
That should be fine.  Maybe place the TODO comment in the else block.